pkg/ruler/notifier.go

```diff
@@ -162,5 +156,16 @@ func amConfigWithSD(rulerConfig *Config, url *url.URL, sdConfig discovery.Config
 		}
 	}
 
+	// Whether to use TLS or not.
+	if rulerConfig.Notifier.TLSEnabled {
```

Comment:
This is a breaking change. Before TLS was enabled just setting the certificates, while now it requires to explicitly enable it. I'm OK-ish introducing this breaking change, but it should be clearly stated in the CHANGELOG as `[CHANGE]`. @hi-rustin can you fix it, please? /cc @dimitarvdimitrov

Reply:
Sorry. I didn't notice it's enabled by default. Then it's not a breaking change.